common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -731,4 +731,87 @@ public void nRingsUnsupported() {
         assertEquals(expected, e.getMessage());
     }
 
+    @Test
```

Comment:
Please also create test cases that have GEOMETRYCOLLECTION(MULTILPOLYGON(), GEOMETRYCOLLECTION())